example-data/turtle/sio-model/birthDateQuality.ttl

```diff
@@ -4,18 +4,18 @@
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
 
-:person1 a obo:NCBITaxon_9606;
-  sio:SIO_000228 :role1;
-  sio:SIO_000217 :birthDateQuality .
+:person_ a sio:SIO_000498 ;
+  sio:SIO_000228 :role_1 ;
+  sio:SIO_000217 :quality_1 .
 
-:birthDateQuality a obo:NCIT_C68615;
-  sio:SIO_000642  :birthDateOutput .
+:quality_1 a obo:NCIT_C68615 ;
```

Comment:
@NuriaQueralt you are using `_1` here is it because you combing birthdatequality turtle file with gender turtle file?

Reply:
Yes. Indeed, it is inherited from the 'personInformation.ttl' I first created from Mark's model, where there are both qualities modeled and related to the same patient. So i needed to distinguish them by labeling quality_1 for birthdate and quality_2 for gender. But, the RDF examples for the qualities in separate files they don't need to be distinguished..

Reply:
I have also made the same correction for patient_status separate status, which are `dead` `alive`cases.